x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_details/monitor_status/monitor_status_panel.tsx

```diff
@@ -64,7 +70,11 @@ export const MonitorStatusPanel = ({
                 xDomain={xDomain}
                 tooltip={{
                   customTooltip: ({ values }) => (
-                    <MonitorStatusCellTooltip timeBin={getTimeBinByXValue(values?.[0]?.datum?.x)} />
+                    <TooltipContainer>
+                      <MonitorStatusCellTooltip
+                        timeBin={getTimeBinByXValue(values?.[0]?.datum?.x)}
+                      />
+                    </TooltipContainer>
```

Comment:
@nickofthyme you can move the container into the `MonitorStatusCellTooltip`

Reply:
Done in 8e9dce4